src/styles/_clock.scss

```diff
@@ -0,0 +1,51 @@
+.clock-position {
+  position: absolute;
+  bottom: 50%;
+  left: 50%;
+  transform: translateX(-50%) rotate(0deg);
+  transform-origin: bottom center;
+  border-radius: 40%;
+}
+
+.clock {
+  &__center {
+    width: 40px;
+    height: 40px;
+    background-color: rgb(129, 118, 12);
+    transform: translate(-50%, 50%);
+    z-index: 100;
+  }
+
+  &__hour-hand {
+    width: 30px;
+    height: 25vh;
+    background-color: rgb(65, 197, 13);
+    z-index: 3;
+    animation: clock 864000s linear infinite;
```

Comment:
move seconds into variables
```
$minute: 60s;
$hour: 60 * $minute;
$half-day: ...;
```

Reply:
also move `animation: clock linear infinite;` to common selector, and change only `animation-name` for hands